### ISSUE TITLE
Feature request : Bold keywords automatically

### ISSUE THREAD

**Issue by gschurck** [sinaatalay/RenderCV#144]
When I send several CVs for different jobs, I like to bold some technical keywords related to the job each time.
But it's a bit cumbersome to do this manually for each CV. So I would like to add an option like this to rendercv :

```yaml
design:
   bold_keywords: 
      - Python
      - FastAPI
      - JavaScript
```

Could it fit into the scope of the project, or should I keep it in a separate fork?
And what might be the best way to implement it: 
- Add asterisks `**` in the yaml before generating latex, but it means using a different yaml than the one in the repo when using ci/cd. But allow to edit a specific word.
- Keep the yaml without the stars `**`, but still make the keywords from the list bold in latex. However, it is not possible to remove the bold from a specific word.
- Add a `bold_keywords` CLI command that we can run manually before generating latex and pdf to bold the corresponding keywords in yaml file each time. Can easily be automated in script, and the use cases are still separate. I think it could be a good tradeoff.

**Comment by sinaatalay** [comment]
This sounds like a really good idea. We should consider adding `rendercv_settings` field in the YAML input file, in addition to `cv`, `design`, and `locale_catalog`. It could contain the `bold_keywords` key.

Although I won't be able to work on new features for some time, I will leave this issue open until it's implemented.

**Comment by tobybaratta** [comment]
I think this can be more easily implemented now, after #146 ! It'd be great to be able to add other formatting guidelines too.

**Comment by sinaatalay** [comment]
This is implemented and will be released in the next version.
